### PR TITLE
New package: cw-4.1.1

### PR DESCRIPTION
**Pull request by akhiljalagam**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by classabbyamp** [comment]
please remove the changes to the repo configs. if you want to change that in your checkout, set XBPS_MIRROR in ./etc/conf

**Comment by wael444** [comment]
uhh, why does this change the mirrors?

File: srcpkgs/cw/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+version=4.1.1
+revision=1
+build_style=go
+go_import_path=github.com/lucagrulla/cw
+short_desc="Best way to tail AWS CloudWatch Logs from your terminal"
+maintainer="akhil <akhil.j@outlook.in>"
+license="Apache-2.0"
+homepage="https://www.lucagrulla.com/cw"
+changelog="https://github.com/lucagrulla/cw/wiki/Release-notes"
+distfiles="https://github.com/lucagrulla/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/cw/template:12]: expand $pkgname here. there's no benefit; it's much longer than the name itself

Comment (akhiljalagam) [reply srcpkgs/cw/template:12]: I checked some other templates and they used it. so i just followed it.

Comment (paper42) [reply srcpkgs/cw/template:12]: let's not add more templates that use it

Comment (akhiljalagam) [reply srcpkgs/cw/template]: done. modified template.

**Comment by wael444** [comment]
could you squash your commits? 
https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#review

**Pull request closed without merging**